api/python/index.md

```diff
@@ -5,7 +5,7 @@ nav_order: 30
 parent: API - Query data programmatically
 has_children: true
 redirect_from:
-    /tutorials/
+    /tutorials
```

Comment:
This is actually the wrong file. It should be in api/python/tutorials.md. AND you need hyphens before the path if there are multiple entries.

Reply:
Fixed- this now redirects to `/api/python/tutorials`